exchanges/order/order_types.go

```diff
@@ -33,7 +33,10 @@ var (
 	ErrNoRates         = errors.New("no rates")
 	ErrCannotLiquidate = errors.New("cannot liquidate position")
 
-	ErrUnknownTrackingMode = errors.New("unknown tracking mode")
+	ErrUnknownTrackingMode   = errors.New("unknown tracking mode")
+	ErrGetOrderRequestFailed = errors.New("failed to get order request")
```

Comment:
🚧 Change request:
* I don't think we can talk about "Requests" in this package.
* But that's okay, because I think it all works without mentioning that it's a request.
```suggestion
	ErrGetFailed = errors.New("failed to get order")
```